### ISSUE TITLE
Lambda * Mu or Lambda / Mu?

### ISSUE THREAD

**Issue by KarahanS** [rachtsy/KPCA_code#5]
Hi, 
in the paper shrinkage operation is given like this:

![Image](https://github.com/user-attachments/assets/59ee745f-a6fd-4671-b3d9-7fee0d6b191e)

In the original RPCA work, it is given as follows:

![Image](https://github.com/user-attachments/assets/9828a44b-52f2-4e52-bf91-0d118420d20a)

with $\lambda \mu$, not $\frac{\lambda}{\mu}$.
I see in your code you do the latter version (multiplication) (https://github.com/rachtsy/KPCA_code/blob/master/Robust/softmax.py see line 58) but in the paper it is the former one (division). Is there a typo in the paper?

**Comment by rachtsy** [comment]
hi KarahanS,

there is no difference if you treat both $\lambda$ and $\mu$ as hyperparamters.
